app/controllers/admin_controller.rb

```diff
@@ -0,0 +1,75 @@
+class AdminController < ApplicationController
+
+  before_action :authenticate_user!
+  before_action :verify_admin
+
+  def verify_admin
+    unless can? :manage, :all
+      redirect_to root_path
+    end
+  end
+
+  def index
+    @users = User.all
+    render layout: "layouts/application"
+  end
+
+  def change_to_admin
+    User.find(user_id).roles = []
```

Comment:
Instead of assigning an empty array, it would look a bit cleaner to user `.clear`